openupgrade_scripts/scripts/website_event/17.0.1.4/pre-migration.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2024 Tecnativa - Pilar Vargas
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from openupgradelib import openupgrade
+
+_xmlids_renames = [
+    (
+        "website_event_questions.constraint_event_registration_answer_value_check",
```

Comment:
```suggestion
        "website_event.constraint_event_registration_answer_value_check",
```
the xmlid changes when the module is merged in `base`. Please, apply the same to the others.

Reply:
Done, thanks!